### PR TITLE
Profile pets

### PR DESCRIPTION
**Pull request by hazelkimberly**
Added loader icons and updated pet display

### CODE REVIEW

**Comment by malloryporche** [comment]
Please review the update functionality without uploading an image.

File: src/components/Profile/AddPets.tsx [hunk 1]
```diff
@@ -31,17 +32,19 @@ export default function AddPets({ profileId }: { profileId: string }) {
   const utils = api.useUtils();
   const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    if (!file) {
-      return;
+
+    let address = "";
```
Comment (redconOne) [inline src/components/Profile/AddPets.tsx:36]: If there is no file does the photoUrl in your mutation send an empty string as an address? 
If so, does that remove a photo that's in place?
Not critiquing, just curious as to how this works.

Comment (redconOne) [reply src/components/Profile/AddPets.tsx:36]: Disregard, I see that you handle that on the backend. Nevermind.

**Review by redconOne — APPROVED** [review]
Looks good! Thanks for all the work on adding pet images!

File: src/components/Profile/AddPets.tsx [hunk 2]
```diff
@@ -31,17 +32,19 @@ export default function AddPets({ profileId }: { profileId: string }) {
...
+
+    if (file) {
+      const filename = `${uuidv4()}`;
+      address = `${env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/images/${filename}`;
+      const { data, error } = await supabase.storage
```
Comment (burhan-syed) [inline src/components/Profile/AddPets.tsx:41]: In the future we should handle the error case

**Review by burhan-syed — APPROVED** [review]

**Pull request merged**